xsnap/makefiles/lin/xsnap-worker.mk

```diff
@@ -144,7 +142,6 @@ $(OBJECTS): $(SRC_DIR)/xsAll.h
 $(OBJECTS): $(SRC_DIR)/xsScript.h
 $(OBJECTS): $(SRC_DIR)/xsSnapshot.h
 $(OBJECTS): $(INC_DIR)/xs.h
-$(OBJECTS): $(EXTRA_DEPS)
```

Comment:
This is an agoric-sdk build mechanism to force rebuild on config changes, please do not remove.